projects/packages/google-analytics/src/class-ga-manager.php

```diff
@@ -284,7 +284,8 @@ public static function amp_analytics_entries( $analytics_entries ) {
 						'request' => 'pageview',
 					),
 				),
-			)
+			),
+			0 // No `json_encode()` flags because we don't want to disrupt the current hash index.
```

Comment:
I think we may need to here. This isn't just calculating a hash, it's also being output somewhere.

Reply:
Hm. I'm not sure the implications of changing it, but at minimum it will mess up the hash index.

* Option 1: Do it anyway.
* Option 2: `wp_json_encode()` the array with no flags for the hash, and with flags for the value.
* Option 3: ?

Reply:
🤷 I have no idea whether the ID means anything, or if it's just a value passed along. I suppose to be safe we could go with option 2.